### DISCUSSION TITLE
Why use showintro=1 for plowe-0 in assets(.dev).json?

### DISCUSSION THREAD

**Discussion by u-RraaLL** [uBlockOrigin/uBlock-issues#2675] [Q&A]
https://pgl.yoyo.org/adservers/serverlist.php?hostformat=hosts&showintro=1&mimetype=plaintext

This is the current contentURL for plowe-0. When there's no `mimetype=plaintext` present, it will show a headers on pgl's website.

The current link didn't work for a while today. So I was unable to update the list when opening a new browser profile or trying an old one.

The list was accessible when the `showintro` value was changed to `0`. How about adding it as a 2nd contentURL since they seem to work independently?

    https://pgl.yoyo.org/adservers/serverlist.php?hostformat=hosts&showintro=0&mimetype=plaintext

**Comment by gorhill** [comment]
That's just the URL I was given when filling the form at <https://pgl.yoyo.org/adservers/>.

**Reply by u-RraaLL** [reply]
I see. What do you think about adding the 2nd link as a backup though?

IDK why the first link didn't work earlier today. Logger showed 500 Internal Server Error and opening the link showed a blank page.  
Changing the `showintro` value to `0` displayed the list just fine, which means one one could work when the other is down.

Or are updates to this list rare enough that a backup is not needed and new uBO installs can just rely on the (currently one month old) copy it ships with?

**Comment by MasterKia** [comment]
@pgl
